src/js/components/DataTable/index.d.ts

```diff
@@ -1,53 +1,92 @@
-import * as React from "react";
-import { 
-  A11yTitleType, 
-  AlignSelfType, 
-  GridAreaType, 
+import * as React from 'react';
+import {
+  A11yTitleType,
+  AlignSelfType,
+  GridAreaType,
   MarginType,
-} from "../../utils";
+  BackgroundType,
+  PadType,
+  BorderType,
+} from '../../utils';
 
-export interface DataTableProps {
+type Sections<T> = { header?: T; body?: T; footer?: T };
+
+type ColumnSizeType =
```

Comment:
Should this be exported so users can use access it directly?

Reply:
Yeah, few of it can be